spanner/tests/system/test_system.py

```diff
@@ -726,6 +726,59 @@ def test_execute_sql_w_query_param(self):
         )
 
 
+class TestStreamingChunking(unittest.TestCase, _TestData):
+
+    @classmethod
+    def setUpClass(cls):
+        from tests.system.utils.streaming_utils import INSTANCE_NAME
+        from tests.system.utils.streaming_utils import DATABASE_NAME
+
+        instance = Config.CLIENT.instance(INSTANCE_NAME)
+        if not instance.exists():
+            raise unittest.SkipTest(
+                "Run 'tests/system/utils/populate_streaming.py' to enable.")
+
+        database = instance.database(DATABASE_NAME)
+        if not instance.exists():
+            raise unittest.SkipTest(
+                "Run 'tests/system/utils/populate_streaming.py' to enable.")
```

Comment:
We've used `self.skipTest` [elsewhere][1]

[1]: https://github.com/GoogleCloudPlatform/google-cloud-python/blob/2d63cda6bcb3a96230d9ec11c59e243b96cc5c20/bigtable/tests/system.py#L133